website/docs/advanced/docker/index.md

```diff
@@ -59,24 +77,13 @@ flowchart LR
 ```
 
 
-The GATK-SV Docker setup is organized as follows:
-
- - **Dockerfile**: 
-   These files define the instructions for building the necessary tools and 
-   configurations required for the GATK-SV pipeline.
-
- - **Docker Images**: Docker images are automatically built based on each Dockerfile. 
-   These images are stored in both Azure Container Registry (ACR) and 
-   Google Cloud Container Registry (GCR). The images serve as self-contained 
-   packages that encapsulate all the tools needed for the GATK-SV pipeline.
-
- - **Docker Containers**: Cromwell, a workflow execution system, creates GATK-SV 
-   Docker containers on virtual machines within the Google Cloud Platform (GCP). 
-   These containers are instantiated based on the Docker images obtained 
-   from GCR. The GATK-SV data analysis tasks are then executed within 
-   these containers, providing a consistent and isolated environment.
 
-In summary, the GATK-SV Docker setup involves multiple Dockerfiles defining 
-the build instructions, resulting in Docker images that are stored in ACR and GCR. 
-These images are used to create Docker containers on GCP virtual machines through Cromwell, 
-where the GATK-SV data analysis takes place.
+Dockerfiles are text files, and GATK-SV stores them on 
+[GitHub](https://github.com/broadinstitute/gatk-sv/tree/main/dockerfiles) 
+for accessibility and version control. 
+Docker images are larger files (e.g., 1GiB) and should be hosted on container registries 
+accessible to runtime environments. GATK-SV stores images on Google Container Registry (GCR) 
+and Azure Container Registry (ACR) so they are accessible to the 
+workflow execution environment on the Terra platform. 
+Docker containers are ephemeral runtime environments, created on 
+virtual machines when the analysis starts, and are “purged” when the analysis finishes.
```

Comment:
Add a note that the builds and resulting GCR and ACR images are identical (in theory)

Reply:
Yes, that would clarify it better.